public/framework/datasources/obs_opensearch_datasource.ts

```diff
@@ -6,25 +6,26 @@
 import { DataSource } from '../../../../../src/plugins/data/public';
 
 interface DataSourceConfig {
+  id: string;
```

Comment:
can we add a list of labels (string array) for future filtering and association support ?